spec/lib/decision_review/service_spec.rb

```diff
@@ -9,231 +9,111 @@
   let(:user) { build(:user, :loa3, ssn: ssn_with_mockdata) }
 
   describe '#post_higher_level_reviews' do
-    context 'with a valid decision review request' do
-      it 'returns an intake status response object' do
-        VCR.use_cassette('decision_review/202_intake_status') do
-          request = {
-            'data' => {
-              'type' => 'HigherLevelReview',
-              'attributes' => {
-                'receiptDate' => '2019-07-10',
-                'informalConference' => true,
-                'sameOffice' => false,
-                'legacyOptInApproved' => true,
-                'benefitType' => 'compensation',
-                'veteran' => {
-                  'fileNumberOrSsn' => '123456789',
-                  'addressLine1' => '123 Street St',
-                  'addressLine2' => 'Apt 4',
-                  'city' => 'Chicago',
-                  'stateProvinceCode' => 'IL',
-                  'zipPostalCode' => '60652',
-                  'phoneNumber' => '4432924565',
-                  'emailAddress' => 'someone@example.com'
-                },
-                'claimant' => {
-                  'participantId' => '44444444',
-                  'payeeCode' => '10'
-                }
-              }
-            },
-            'included' => [
-              {
-                'type' => 'RequestIssue',
-                'attributes' => {
-                  'decisionIssueId' => 2
-                }
-              }
-            ]
-          }
-
-          response = subject.post_higher_level_reviews(request.to_json)
-          expect(response).to be_accepted
-          expect(response).to be_an DecisionReview::Responses::Response
+    subject { described_class.new.post_higher_level_reviews(body: body.to_json, user: user) }
+    let(:body) { VetsJsonSchema::EXAMPLES['HLR-CREATE-REQUEST-BODY'] }
```

Comment:
YES! I really like using shared examples between BE & FE!

Reply:
https://emojis.slackmojis.com/emojis/images/1547582922/5197/party_blob.gif